draft-ietf-httpbis-unprompted-auth.md

```diff
@@ -349,7 +349,7 @@ its key ID.
 ## The s Parameter {#parameter-s}
 
 The REQUIRED "s" (signature) parameter is an integer that specifies the
-signature scheme used to compute the proof transmitted in the "p" directive.
+signature scheme used to compute the proof transmitted in the `p` Parameter.
```

Comment:
Is there a reason Parameter is capitalized here, but not above?

Reply:
fixed